docs/specification.md

```diff
@@ -614,37 +612,16 @@ Parameter (in Michelson):
 )
 ```
 
-- The proposal is saved and assigned a unique numeric proposal ID.
+- The proposal is saved under `BLAKE2b` hash of proposal value.
```

Comment:
Note that we hash a pair where `SENDER` is included.

Reply:
Oh, good catch.